types/test.ts

```diff
@@ -1,7 +1,6 @@
-import setupFm, { MockResponseInit } from 'vitest-fetch-mock';
-import {vi} from 'vitest';
+import { MockResponseInit, createFetchMock } from 'vitest-fetch-mock';
 
-const fm = setupFm(vi);
+const fetchMock = createFetchMock();
```

Comment:
Required: this is a pretty big change to the API that I don't think we discussed.  I'd prefer to keep the API unchanged in this PR, so I can think about / review it separately.